lib/components/SideMenu/fixed-array.ts

```diff
@@ -0,0 +1,188 @@
+export let fixedMenuItemsList =
```

Comment:
rename this file so it's clear that it is used in tests, e.g. `fixed-array.mock.ts`